src/sas/sascalc/pr/invertor.py

```diff
@@ -57,7 +55,8 @@ def help():
 
 class Invertor():
 
-    def __init__(self, logic: InversionLogic):
+    # "InversionLogic" type hint for "logic" removed to avoid circular import
+    def __init__(self, logic):
```

Comment:
We should keep type hints in most cases. To get around circular imports, and only use the import for type checking sake, you can try:

```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from sas.qtgui.Perspectives.Inversion.InversionLogic import InversionLogic
```